File: Codeforces/C++/problemset/Petya and Strings.cpp
// Problem Link: https://codeforces.com/problemset/problem/112/A

#include <iostream>
#include <string>
using namespace std;
int main()
{
    string x,y;
    cin >> x >> y;
    for(int i = 0; i < x.size(); i++)
    {
        x[i] = toupper(x[i]);
        y[i] = toupper(y[i]);
        if(x[i] > y[i])
        {
            cout << 1;
            return 0;
        }

        else if(x[i] < y[i])
        {
            cout << -1;
            return 0;
        }
    }
    cout << 0;

    return 0;
}

File: src/main/kotlin/PetyaAndStrings.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/112/A */

fun main() {
    val s1 = readln()
    val s2 = readln()

    for (i in s1.indices) {
        if(s1[i].lowercase() > s2[i].lowercase()) {
            print(1)
            return
        } else if(s1[i].lowercase() < s2[i].lowercase()) {
            print(-1)
            return
        }
    }

    print(0)
}